src/Util/AstHelper.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Util;
+
+use Closure;
+use PhpParser\Node;
+
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Reflection\Reflection;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflection\ReflectionFunction;
+
+use PhpParser\Node\Expr\Closure as ClosureNode;
+use PhpParser\Node\Expr\ClosureUse as ClosureUseNode;
+use PhpParser\Node\Expr\ArrowFunction as ArrowFunctionNode;
+
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor\FindingVisitor;
+use PhpParser\PrettyPrinter\Standard as StandardPrettyPrinter;
+use PhpParser\PrettyPrinterAbstract;
+use RuntimeException;
+
+final class AstHelper {
+    private BetterReflection $betterReflection;
+
+    public function __construct( ?BetterReflection $betterReflection = null ) {
+        if ( !$betterReflection  ) {
+            $this->betterReflection = new BetterReflection();
+        } else {
+            $this->betterReflection = $betterReflection;
+        }
+    }
+    protected function getBetterReflection(): BetterReflection {
+        return $this->betterReflection;
+    }
+    /**
+     * 
+     * @return Node[]|null $nodes
+     * 
+     */
+    public function getAstForReflection(ReflectionClass|ReflectionFunction $reflection): array|null {
+        $parser = $this->getBetterReflection()->phpParser();
+        $source = $reflection->getLocatedSource();
+        return $parser->parse($source->getSource());
```

Comment:
What happens if the `LocatedSource` contains multiple symbols?

For example:

```php
function foo() {}
function bar() {}
```

We probably only want the AST of the requested symbol, no?